imgui_draw.cpp

```diff
@@ -893,6 +893,26 @@ void ImDrawList::PathArcTo(const ImVec2& center, float radius, float a_min, floa
     }
 }
 
+void ImDrawList::PathEllipticalArcTo(const ImVec2& center, float radius_x, float radius_y, float rot, float a_min, float a_max, int num_segments)
+{
+    _Path.reserve(_Path.Size + (num_segments + 1));
+
+    const float cosRot = ImCos(rot);
+    const float sinRot = ImSin(rot);
+    for(int i = 0; i <= num_segments; i++)
+    {
```

Comment:
I can alter the commit later but if you are going to rewrite this commit, `cosRot` --> `cos_rot` and spacing after `for`. Otherwise I'll fix them later.